code/processes/wdb.q

```diff
@@ -21,11 +21,11 @@ mode:@[value;`mode;`saveandsort];	/- the wdb process can operate in three modes
 									/- 						data on disk, apply attributes and the trigger a reload on the
 									/-						rdb and hdb processes
 
-writedownmode:@[value;`writedownmode;`default];			/- the wdb process can periodically write data to disc and sort at EOD in two ways:
-														/- 1. default 		- 	the data is partitioned by [ partitiontype ]
-														/-						at EOD the data will be sorted and given attributes according to sort.csv before being moved to hdb
-														/- 2. partbyattr	-	the data is partitioned by [ partitiontype ] and the column(s) assigned the parted attributed in sort.csv
-														/-						at EOD the data will be merged from each partiton before being moved to hdb									
+writedownmode:@[value;`writedownmode;`default];			        /- the wdb process can periodically write data to disc and sort at EOD in two ways:
+									/- 1. default                   -       the data is partitioned by [ partitiontype ]
+									/-                                      at EOD the data will be sorted and given attributes according to sort.csv before being moved to hdb
+                                                                        /- 2. partbyattr                -       the data is partitioned by [ partitiontype ] and the column(s) assigned the parted attributed in sort.csv
+                                                                        /-                                      at EOD the data will be merged from each partiton before being moved to hdb									
```

Comment:
Misaligned comment here on 27.

Reply:
yeah, don't know what happened with those. it's the second time I am fixing them :S